libmolsketch/atom.cpp

```diff
@@ -985,19 +985,19 @@ namespace Molsketch {
     QPointF intersection;
 
     QLineF topEdge{bounds.topLeft(), bounds.topRight()};
-    if (topEdge.intersects(line, &intersection) == QLineF::BoundedIntersection)
+    if (topEdge.intersect(line, &intersection) == QLineF::BoundedIntersection)
```

Comment:
Hm... that's unfortunately going to break compatibility with Qt >= 5.14. Yeah, they really just renamed that function apparently. I'm afraid we'll need a macro here. It would be super nice if you could define something like
```
#if QT_VERSION >= QT_VERSION_CHECK(5,14,0)
#define INTERSECT intersects
#else
#define INTERSECT intersect
#endif
```
and then use e.g. `topEdge.INTERSECT(line, &intersection)`.
But I don't insist... ;)